src/test/resources/application.conf

```diff
@@ -1,20 +1,26 @@
 slack {
   webhook {
-    url = "http://localhost:9002/webhook"
+    url = "aHR0cDovL2xvY2FsaG9zdDo5MDAyL3dlYmhvb2s=" # http://localhost:9002/webhook
```

Comment:
Is it worth adding some additional information to the comment to explain its the decoded string?